### ISSUE TITLE
upgrade_check exits with 1 on success (when using with '-i' command line option)

### ISSUE THREAD

**Issue by davidshao** [apache/airflow#14858]
**Apache Airflow version**:
1.10.14

**Kubernetes version (if you are using kubernetes)** (use `kubectl version`):
n/a

**Environment**:

- **Cloud provider or hardware configuration**:
- **OS** (e.g. from /etc/os-release):
```
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```
- **Kernel** (e.g. `uname -a`):
- `Linux iebdev22 3.10.0-1160.11.1.el7.x86_64 #1 SMP Fri Dec 18 16:34:56 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux`
- **Install tools**:
- **Others**:

**What happened**:

When running `upgrade_check` (v1.3.0) with `-i` (ignore) option, it exits with `1` on success.

**What you expected to happen**:

`upgrade_check` should exit with 0 on success.

**How to reproduce it**:

```
(ve) $ airflow upgrade_check --version
1.3.0
```

```
 (ve)$ airflow upgrade_check -i VersionCheckRule -i PodTemplateFileRule

=================================== STATUS =============================================

Remove airflow.AirflowMacroPlugin class..........................................SUCCESS
Ensure users are not using custom metaclasses in custom operators................SUCCESS
Chain between DAG and operator not allowed.......................................SUCCESS
Connection.conn_type is not nullable.............................................SUCCESS
Custom Executors now require full path...........................................SUCCESS
Check versions of PostgreSQL, MySQL, and SQLite to ease upgrade to Airflow 2.0...SUCCESS
Hooks that run DB functions must inherit from DBApiHook..........................SUCCESS
Fernet is enabled by default.....................................................SUCCESS
GCP service account key deprecation..............................................SUCCESS
Unify hostname_callable option in core section...................................SUCCESS
Changes in import paths of hooks, operators, sensors and others..................SUCCESS
Legacy UI is deprecated by default...............................................SUCCESS
Logging configuration has been moved to new section..............................SUCCESS
Removal of Mesos Executor........................................................SUCCESS
No additional argument allowed in BaseOperator...................................SUCCESS
SendGrid email uses old airflow.contrib module...................................SUCCESS
Check Spark JDBC Operator default connection name................................SUCCESS
Changes in import path of remote task handlers...................................SUCCESS
Connection.conn_id is not unique.................................................SUCCESS
Use CustomSQLAInterface instead of SQLAInterface for custom data models..........SUCCESS
Found 0 problems.

Not found any problems. World is beautiful.
You can safely update Airflow to the new version.

(ve)$ echo $?
1
```

**Anything else we need to know**:

**Comment by mcepok** [comment]
Sounds like the issue I fixed in #14740.

**Comment by mcepok** [comment]
@kaxil I think this one can closed because of #14740 .

**Comment by kaxil** [comment]
closed by https://github.com/apache/airflow/pull/14740

Thanks @mcepok
